_data/hipaa-apps.yml

```diff
@@ -0,0 +1,57 @@
+    available: "yes"
+
+  - title: Logging
+    description: Apps that are Dockerized should be able to “just” send logs to stdout/stderr and have everything work as intended, but this is a vast over-simplification. First, not all frameworks send logs to stdout/stderr by default. Second, not everyone uses Docker. Third, even with Docker apps may require sidecars and proper configs to aggregate or ship logs properly.
+    available: "yes"
+
+  - title: Packaging
+    description: Knowing how to Dockerize an app properly is surprisingly hard. Developers must balance small base images vs secure/up-to-date base images, be thoughtful about use of the Docker Layer Cache, how to version and where to store artifacts, to name a few concerns.
+    available: "yes"
+
+  - title: Secrets management
+    description: Apps may use an external service to store or encrypt secrets (e.g., AWS Secrets Manager, Vault, KMS, K8S), however they still must do the right thing to use that secret.
+    available: "yes"
+
+  - title: App configuration
+    description: Apps must be configurable to run differently in different environments (dev, stage, prod, etc). They may use config files in JSON, YAML, or a thousand other formats. Configuration involves careful treatment of sensitive values, hot reloading to prevent downtime, exposing performance tuning settings, and contstraints unique to the organization deploying the application.
+    available: "yes"
+
+  - title: Database management
+    description: Apps require schema migrations using tools like Flyway, ActiveRecord, Phinx, etc. These may need to be run before the app boots or as a separate deploy step. Secure database connections, connection pooling, TLS usage, certificate validation, service discovery, and ORM or DB library usage must all be thought through and carefully implemented.
+    available: "yes"
+
+  - title: Server hardening
+    description: Apps running on EC2 must handle or consider security and compliance requirements such as firewalls, anti-virus, file integrity monitoring, SSH access, VPN access, automatic updates, time synchronization and more.
+    available: "yes"
+
+  - title: Build system
+    description: Frameworks all have a build system that you have to configure in order to properly manage dependencies, compilation, tests and more.
+    available: "yes"
+
+  - title: Automated tests
+    description: All maintainable applications need good tests, but setting them up correctly requires knowledge of pre-commit checks, code-formatting, static analysis, unit tests, integration tests, end to end tests, performance tests, testing frameworks, clear test output and more.
+    available: "yes"
+
+  - title: Asset pipeline
+    description: Applications that depend on static content typically require asset compilation (e.g., compile TypeScript to JS), minification, concatenation, fingerprinting, deployment to one or more CDNs, and making the URLs work in dev & prod.
+
+    available: "yes"
```

Comment:
I don't think our sample app does any asset pipeline stuff yet

Reply:
Changed to "Coming soon" for continuity with the index page's table in c165964.